### PR TITLE
fix(plugins): omit launch paths when unused [LIBS-477]

### PR DESCRIPTION
**Pull request by KaiVandivier**
Addresses [LIBS-477](https://dhis2.atlassian.net/browse/LIBS-478), and also sneaks in the one-line fix for [LIBS-355](https://dhis2.atlassian.net/browse/LIBS-355)

Fixing [LIBS-479](https://dhis2.atlassian.net/browse/LIBS-479) is not so straightforward because of how the plugin builds depends on the app build, and needs some more work

[LIBS-478]: https://dhis2.atlassian.net/browse/LIBS-478?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIBS-355]: https://dhis2.atlassian.net/browse/LIBS-355?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIBS-479]: https://dhis2.atlassian.net/browse/LIBS-479?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LIBS-477]: https://dhis2.atlassian.net/browse/LIBS-477?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: cli/src/lib/generateManifests.js [hunk 1]
```diff
@@ -93,11 +115,20 @@ module.exports = (paths, config, publicUrl) => {
             },
         ],
         start_url: '.',
-        display: 'browser',
+        display: 'standalone',
```
Comment (amcgee) [inline cli/src/lib/generateManifests.js:118]: What's the impact of this change?  Seems a bit unrelated?

**Review by amcgee — APPROVED** [review]
One comment but looks good 👍

File: cli/src/lib/generateManifests.js (hunk 1 above)
Comment (KaiVandivier) [reply cli/src/lib/generateManifests.js:118]: It is unrelated, but easy to fix - see https://dhis2.atlassian.net/browse/LIBS-355
